_flake8_tergeo/checks/ast_raise.py

```diff
@@ -50,3 +51,45 @@ def _check_raise_from_itself(node: ast.Raise) -> IssueGenerator:
             issue_number="044",
             message="Found exception raised from itself.",
         )
+
+
+def _get_except(node: ast.AST) -> ast.ExceptHandler | None:
+parent: ast.AST | None = node
+while parent:
+    if isinstance(parent, ast.ExceptHandler):
+        break
+    parent = get_parent(parrent)
+return parent
```

Comment:
My suggestion didnt include the correct indentation, sorry.

Reply:
already fixed 😄